README.md

```diff
@@ -81,11 +81,56 @@ Examples:
 #### Manual
 Download the latest release from the [releases page](https://github.com/Abdenasser/neohtop/releases).
 
-#### Brew
+#### Package Managers
```

Comment:
can we mention something like:

Members of the community have kindly published unofficial packages for various platforms and package managers.

and maybe add a note like:

Important Note
These packages are community-maintained and not officially released, reviewed, or endorsed by the project.
We only provide official builds through the GitHub Releases page.
Since these external packages are managed by third parties, we cannot guarantee their security, integrity, or update frequency. Please use them at your own discretion.

Reply:
thank you @Owen-sz